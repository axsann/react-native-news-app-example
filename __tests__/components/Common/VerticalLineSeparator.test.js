// @flow

import 'react-native'
import React from 'react'
import VerticalLineSeparator from '../../../src/components/Common/VerticalLineSeparator.component'

// Note: test renderer must be required after react-native.
import renderer from 'react-test-renderer'

describe('VerticalLineSeparator', () => {
  it('renders correctly', () => {
    const instance = renderer.create(
      <VerticalLineSeparator />
    )

    expect(instance.toJSON()).toMatchSnapshot()
  })
})
