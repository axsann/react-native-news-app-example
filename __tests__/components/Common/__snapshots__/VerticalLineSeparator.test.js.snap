// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VerticalLineSeparator renders correctly 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#eeeeee",
      "width": 1,
    }
  }
/>
`;
